milestone2.ts

```diff
@@ -83,34 +83,75 @@ const userData: Volunteer[] = [
   Hint: user data is stored in the userData object above. 
 */
 type GetNumber = (data: Volunteer[]) => number;
-let findAverage: GetNumber; // Define lambda function here
+let findAverage: GetNumber = (data)=>{
+  var total = 0;
+  for(var i=0; i<data.length;i++)
+  {
+    total= total+data[i].age;
+  }
+  return total/data.length
+  
+}// Define lambda function here
 
-//console.log(findAverage(userData));
+console.log(findAverage(userData));
 
 // Question 2: Data Handling
 /* Use data handling function(s) to find the first index of someone from San Francisco (SF).
   Return -1 if no one is from San Francisco.
   Hint: Use a lambda function as a value
 */
-let findIndexAns: GetNumber; // Code here
+let findIndexAns: GetNumber = (data)=>{
+  for(var i=0; i<data.length;i++)
+  {
+    if(data[i].city==="SF")
+    {
+      return i
+    }
+  }
+  return -1
+}// Code here
```

Comment:
Please update your code for questions 2-4 to use the data handling functions described [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array). Each data handling function should each take a function as an argument (e.g. map, find, filter, etc). Sorry for the mix up!